paasta_tools/kubernetes_tools.py

```diff
@@ -1588,6 +1588,8 @@ def get_kubernetes_metadata(self, git_sha: str) -> V1ObjectMeta:
                 "paasta.yelp.com/service": self.get_service(),
                 "paasta.yelp.com/instance": self.get_instance(),
                 "paasta.yelp.com/git_sha": git_sha,
+                "paasta.yelp.com/pool": self.get_pool(),
```

Comment:
we probably want to start switching to using `paasta_prefixed()` where we can :)